src/main/resources/api.yaml

```diff
@@ -119,6 +119,20 @@ paths:
             type: array
             items:
               $ref: '#/definitions/SprintDto'
+  /projects/{id}/sprints/{id}/close:
```

Comment:
Nie podoba mi się to. PUT służy do aktualizowania całego obiektu (w tym przypadku sprintu), więc wprowadzenie dodatkowego adresu `/close`, na który będzie wysyłany cały sprint (a aktualizowany tylko status) chyba nie jest dobrym pomysłem.

Proponowałbym albo PUT bez `/close`, albo PATCH
https://stackoverflow.com/a/45688487

@jplewa @kjarosh @ktokolwiek ?

Reply:
> Proponowałbym albo PUT bez /close, albo PATCH
Zgadzam się. 

`PATCH` brzmi mądrze w sumie

Reply:
nie wysylamy calego sprintdto tylko pusty request
zrobie jako patch, zastanawialem sie nad tym ale pomyslalem ze na razie duzo nam to nie da, ale, niech juz bedzie porzadnie od poczatku

Reply:
Ups, jak pisałem, to nie podświetliło xd

Reply:
Ja zrobiłam to samo z @ TeamFrontend kiedyś!